packages/loaders/src/Loader.ts

```diff
@@ -155,6 +155,15 @@ export class Loader extends ResourceLoader
 
         return Loader;
     }
+
+    /**
+     * All the resources for this loader keyed by name.
+     *
+     * @member {object<string, ILoaderResource>}
```

Comment:
I know `PIXI.ILoaderResource` isn't documented, but all of the API is under `PIXI`. Maybe we should add docs to `PIXI.ILoaderResource` @bigtimebuddy?

```suggestion
     * @member {object<string, PIXI.ILoaderResource>}
```